pkg/k8s/events/types.go

```diff
@@ -30,7 +39,145 @@ const (
 
 // PubSubMessage represents a common messages abstraction to pass through the PubSub interface
 type PubSubMessage struct {
-	Kind   announcements.Kind
+	Kind   Kind
+	Type   EventType
 	OldObj interface{}
 	NewObj interface{}
 }
+
```

Comment:
Are these event related changes accidentally included in this PR?

Reply:
Those got caught in here somehow. I'll  fix once https://github.com/openservicemesh/osm/pull/4939 is submitted